src/test/java/org/openrewrite/java/testing/jmockit/JMockitNonStrictExpectationsToMockitoTest.java

```diff
@@ -0,0 +1,1097 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.jmockit;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class JMockitNonStrictExpectationsToMockitoTest implements RewriteTest {
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion()
+            .logCompilationWarningsAndErrors(true)
+            .classpathFromResources(new InMemoryExecutionContext(),
+              "junit-4.13.2",
+              "jmockit-1.22",
+              "mockito-core-3.12",
+              "mockito-junit-jupiter-3.12"
```

Comment:
I will remove this line, not needed, as doing legacy JMockit migration which will be using junit 4 or lower

Reply:
Shouldn't this be compatible with both JUnit 4 and 5?

Reply:
For NonStrictExpectations, they would be using junit 4 or older as NonStrictExpectations is not there in jmockit 1.23 upwards. Last version was 1.22. Hence, would like to support this situation.

Reply:
I believe there's nothing preventing the usage of older JMockit versions with JUnit 5, so in my opinion we should support JUnit 5 migration as well.

JUnit 5 documentation seems to imply there are no compatibility concerns: https://jmockit.github.io/tutorial/Introduction.html#:~:text=JMockit%20supports%20(and%20requires)%20the,%2Fjmockit.